dataherald/db_scanner/services/big_query_scanner.py

```diff
@@ -17,26 +17,16 @@
 class BigQueryScanner(AbstractScanner):
     @override
     def cardinality_values(self, column: Column, db_engine: SQLDatabase) -> list | None:
-        if "ARRAY" in str(column.type).upper():
-            count_query = f"SELECT COUNT(DISTINCT element) FROM {column.table.name}, UNNEST({column.name}) as element"  # noqa: S608
-        else:
-            count_query = f"SELECT APPROX_COUNT_DISTINCT({column.name}) FROM {column.table.name}"  # noqa: S608
-
+        count_query = f"SELECT APPROX_COUNT_DISTINCT({column.name}) FROM {column.table.name}"  # noqa: S608
```

Comment:
You can set the try catch here
`        try:
            rs = db_engine.engine.execute(
                f"SELECT APPROX_COUNT_DISTINCT({column.name}) FROM {column.table.name}"  # noqa: S608 E501
            ).fetchall()
        except DatabaseError:
            return None
`